channeldb/channel.go

```diff
@@ -2188,6 +2180,39 @@ func (c *OpenChannel) AdvanceCommitChainTail(fwdPkg *FwdPkg) error {
 			return err
 		}
 
+		// Persist the unsigned acked updates that are not included
+		// in their new commitment.
+		updateBytes := chanBucket.Get(unsignedAckedUpdatesKey)
+		if updateBytes == nil {
+			newRemoteCommit = &newCommit.Commitment
+			return nil
+		}
+
+		r := bytes.NewReader(updateBytes)
+		unsignedUpdates, err := deserializeLogUpdates(r)
+
+		var validUpdates []LogUpdate
+
+		for _, upd := range unsignedUpdates {
+			lIdx := upd.LogIndex
+
+			// Check that it's not on their remote commitment
```

Comment:
perhaps more clear, "Filter for updates that are still on the remote commitment."?

Reply:
done